src/components/Article.jsx

```diff
@@ -1,20 +1,46 @@
-import {useEffect, useState} from 'react';
+import {useEffect, useState, useContext} from 'react';
 import {useParams} from 'react-router-dom';
+import { patchArticles } from '../utils/api';
 
 export default function Article() {
     const {article_id} = useParams();
     const [currArticle, setCurrArticle] = useState({});
-
+    const [isLoading, setIsLoading] = useState(true);
+    
+    
     useEffect(() => {
         fetch(`https://news-app-backend123.herokuapp.com/api/articles/${article_id}`)
         .then((response) => {
             return response.json();
         })
         .then((data) => {
             setCurrArticle(data.article);
+            setIsLoading(false)
         });
     }, [article_id]);
 
+    const handleVote = (article_id, vote) => {
+        if(vote === 1){
```

Comment:
if statement isn't completely necessary as if you. + 1 it'll add one but if you. +-1 it will minus 1